lib/repl.js

```diff
@@ -928,10 +936,15 @@ REPLServer.prototype.convertToContext = function(cmd) {
 
 // If the error is that we've unexpectedly ended the input,
 // then let the user try to recover by adding more input.
-function isRecoverableError(e) {
-  return e &&
-      e.name === 'SyntaxError' &&
-      /^(Unexpected end of input|Unexpected token :)/.test(e.message);
+function isRecoverableError(e, self) {
+  if (e && e.name === 'SyntaxError') {
+    if (e.message == 'Unterminated template literal') {
```

Comment:
there are actually a few different messages you could get, `unterminated_template` and `unterminated_template_expr` in particular would affect this